File: Gaji.py
print("----------------------------------")
print("PROGRAM GAJI KARYAWAN")
print("----------------------------------")

nama = str(input("Masukkan Nama = "))
giliran = int(input("Masukkkan Giliran = "))
Jam = int(input("Masukkan Jam Kerja ="))

if giliran == 1:
        gapok = 2500000
        Jam = 340000
elif giliran == 2:
        gapok = 3850000
        Jam = 470000
elif giliran == 3:
        gapok = 4500000
        Jam = 690000
elif giliran == 4:
        gapok = 5000000
        Jam = 750000
elif giliran == 5:
        gapok = 5700000
        Jam = 830000
else:
        gapok = 0
        Jam = 0
        
if Jam > 10:
        Lembur = Jam - 10
        JamLembur = Lembur * 450000
else:
        JamLembur = 0
        
gajiBersih = Lembur + Jam + gapok

print("=====================================")
print("Nama Karyawan            : ", nama)
print("Gaji Pokok 1 Bulan       : Rp ", gapok)
print("Gaji Lembur              : Rp ", Lembur)
print("Tunjangan Lama Kerja     : Rp ", Jam)
print("Gaji Bersih              : Rp ",gajiBersih)
print("=====================================")

File: Gaji/JAVAVSC/Gaji.java
package JAVAVSC;
import java.util.Scanner;

public class Gaji{
        public static void main(String[] args)
        {
                Scanner input = new Scanner(System.in);

                int jamKerja, giliran, Lembur, lamaKerja, gajiPokok, jamLembur;
                float gajiBersih;
                String namaKaryawan;

                System.out.println("\nMasukkan Nama : ");
                namaKaryawan = input.nextLine();
                System.out.println("\nMasukkan Giliran : ");
                giliran = input.nextInt();
                System.out.println("\nMasukkan Jam Kerja : ");
                jamKerja = input.nextInt();

                if (jamKerja > 10){
                        jamLembur = jamKerja-10;
                }else{
                        jamLembur = 0;
                }

                switch(giliran){
                        case 1:
                                gajiPokok = 2500000;
                                lamaKerja = 340000;
                                break;
                        case 2:
                                gajiPokok = 3850000;
                                lamaKerja = 470000;
                                break;
                        case 3:
                                gajiPokok = 4500000;
                                lamaKerja = 690000;
                                break;
                        default :
                                gajiPokok = 0;
                                lamaKerja = 0;
                                break;
                }
                Lembur=jamLembur*45000;
                gajiBersih=(Lembur+gajiPokok+lamaKerja);

                System.out.println("\nNama Karyawan : "+namaKaryawan);
                System.out.println("\tGaji Pokok 1 Bulan : "+gajiPokok);
                System.out.println("\tGaji Lembur : "+Lembur);
                System.out.println("\tTunjangan Lama Kerja : "+lamaKerja);
                System.out.println("\tGaji Bersih : "+gajiBersih);

                input.close();
        }
}